### PR TITLE
Refactor Login Page and added new test suite and test cases to run wi…

### PR DESCRIPTION
**Pull request by sibesp**
Refactor Login Page and added new test suite and test cases to run with gradle

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -38,17 +43,19 @@ task runTestNGTestCases(type: Test) {
...
     useTestNG()
 
-    testLogging {
-        events "started", "passed", "failed", "skipped"
-    }
+
+
+    //testLogging {
+    //    events "started", "passed", "failed", "skipped"
+    //}
```
Comment (carledriss) [inline build.gradle:50]: Remove commented lines

File: build.gradle [hunk 2]
```diff
@@ -38,17 +43,19 @@ task runTestNGTestCases(type: Test) {
...
 
     options {
-        suites(suites)
+        suites('src/test/resources/Suite1.xml')
```
Comment (carledriss) [inline build.gradle:53]: This is hardcoded
It should exist a way to execute the by suites something like this

```
gradle executeTC -Psuites=Suite1.xml,Suite2.xml
```

File: src/main/java/com/selenium/course/framework/WebDriverManager.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+
+
+public class WebDriverManager {
+
+    private static WebDriverManager instance;
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    private final String baseUrl = "https://login.salesforce.com/";
```
Comment (carledriss) [inline src/main/java/com/selenium/course/framework/WebDriverManager.java:27]: This is hardcoded
Added them to gradle.properties

File: src/main/java/com/selenium/course/framework/WebDriverManager.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    private final int timeout_normal = Environment.getInstance().getTimeOut();
+    private final String browser = Environment.getInstance().getBrowser();
+    private final String mode = Environment.getInstance().getMode();
+    private final String sauceUser = "carledriss";
+    private final String sauceAccessKey = "82f05dc5-b331-46b0-8781-06dd0483088e";
```
Comment (carledriss) [inline src/main/java/com/selenium/course/framework/WebDriverManager.java:32]: These are hardcoded
Added them to gradle.properties

File: src/main/java/com/selenium/course/framework/WebDriverManager.java [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+                );
+                driver = new PhantomJSDriver(caps);
+            }
+        } else if (mode.equals("Remote")) {
+            // Choose the browser, version, and platform to test
+            DesiredCapabilities caps = DesiredCapabilities.iphone();
+            caps.setCapability("platform", "OS X 10.10");
+            caps.setCapability("version", "9.0");
+            caps.setCapability("deviceName","iPad Simulator");
+            caps.setCapability("deviceOrientation", "portrait");
```
Comment (carledriss) [inline src/main/java/com/selenium/course/framework/WebDriverManager.java:79]: These are hardcoded
Added them to gradle.properties

File: src/main/java/com/selenium/course/pages/ContactForm.java [hunk 6]
```diff
@@ -162,8 +162,8 @@
 
     public ContactForm() {
         //this.driver = driver;
-        wait = DriverManager.getInstance().getWait();
-        driver = DriverManager.getInstance().getDriver();
+        wait = WebDriverManager.getInstance().getWait();
+        driver = WebDriverManager.getInstance().getDriver();
         PageFactory.initElements(driver, this);
         try {
             wait.withTimeout(3, TimeUnit.SECONDS).until(
```
Comment (carledriss) [inline src/main/java/com/selenium/course/pages/ContactForm.java:169]: This is hardcoded
Use a constant for that
something like this

```
wait.withTimeout(TIMEOUT_MIN, TimeUnit.SECONDS).until(
```

**Comment by sibesp** [comment]
It is not my part hehehehehe I just was uploading the code base and a version of code that compiles. This will be edited by another team member

**Pull request merged**